docs/ruleset_functions/get_SHW_types_and_spaces.md

```diff
@@ -0,0 +1,32 @@
+
+## get_SHW_types_and_spaces
+
+Description: This function gets all the spaces and the SHW space types and sorts them into a dictionary where the keys are the ServiceWaterHeatingSpaceOptions2019ASHRAE901 and values are a list if space.ids  
+
+Inputs:
+- **RMD**
+
+Returns:
+- **shw_and_spaces_dict**: A dictionary containing where the keys are the ServiceWaterHeatingSpaceOptions2019ASHRAE901 in the RMD and values are lists of space ids.  Example:  
+{"DORMITORY":["sp1","sp2","sp3"], "AUTOMOTIVE_FACILITY":["g1","g2","g3"]}
+
+Function Call:
+
+- 
+
+Data Lookup: None
+
+Logic:
+
+- create a blank dictionary: `shw_and_spaces_dict = {}`
+- look at each space in the RMD: `for space in RMD...spaces:`
+    - get the shw heating space type: `shw_heating_space_type = space.service_water_heating_space_type`
```

Comment:
I wonder if we should look at the buildingsegment.lighting_building_area_type data element as well. I am not sure this will be defined for many projects.

Reply:
Agreed, we provide mapping from lighting building area type to some of the other building area types. This seems like another good candidate for the same approach